pkg/repository/git.go

```diff
@@ -33,29 +34,47 @@ func newGitRepoController() gitRepoController {
 	}
 }
 
+type cleanStatus struct {
+	IsClean bool
+	Err     error
```

Comment:
I think we can keep fields private

Reply:
done in https://github.com/okteto/okteto/pull/3648/commits/d61f715e597663ba5a3903089869e803400b4277